website/content/en/docs/kubebuilder/references/logging.md

```diff
@@ -0,0 +1,261 @@
+{"level":"info","ts":1587741740.407855,"logger":"scoped","msg":"Printing at INFO level"}
+```
+
+#### Output overriding the log level to 1 (debug)
+```console
+$ go run main.go --zap-log-level=debug
+INFO[0000] Running the operator locally in namespace default. 
+{"level":"info","ts":1587741837.602911,"logger":"global","msg":"Printing at INFO level"}
+{"level":"debug","ts":1587741837.602964,"logger":"global","msg":"Printing at DEBUG level"}
+{"level":"info","ts":1587741837.6029708,"logger":"scoped","msg":"Printing at INFO level"}
+{"level":"debug","ts":1587741837.602973,"logger":"scoped","msg":"Printing at DEBUG level"}
+```
+## Custom zap logger
+
+In order to use a custom zap logger, [`zap`][controller_runtime_zap] from controller-runtime can be utilized to wrap it in a `logr` implementation.
+
+Below is an example illustrating the use of [`zap-logfmt`][logfmt_repo] in logging.
+
+### Example
+
+In your `main.go` file, replace the current implementation for logs inside the `main` function:
+
+```Go
+...
+// Add the zap logger flag set to the CLI. The flag set must
+// be added before calling flag.Parse().
+	opts := zap.Options{}
+	opts.BindFlags(flag.CommandLine)
+	flag.Parse()
+
+	logger := zap.New(zap.UseFlagOptions(&opts))
+	logf.SetLogger(logger)
+...
+```
+
+With:
+
+```Go
+	import(
+	...
+	zaplogfmt "github.com/sykesm/zap-logfmt"
+	uzap "go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	...
+)
+	configLog := uzap.NewProductionEncoderConfig()
+	configLog.EncodeTime = func(ts time.Time, encoder zapcore.PrimitiveArrayEncoder) {
+		encoder.AppendString(ts.UTC().Format(time.RFC3339Nano))
+	}
+	logfmtEncoder := zaplogfmt.NewEncoder(configLog)
+
+	// Construct a new logr.logger.
+	logger := zap.New(zap.UseDevMode(true), zap.WriteTo(os.Stdout), zap.Encoder(logfmtEncoder))
+	logf.SetLogger(logger)
+```
+
+**NOTE**: For this example, you will need to add the module `"github.com/sykesm/zap-logfmt"` to your project. Run `go get -u github.com/sykesm/zap-logfmt`.
+
+#### Output using custom zap logger
+
+```console
+$ go run main.go
+ts=2020-04-30T20:35:59.551268Z level=info logger=global msg="Printing at INFO level"
+ts=2020-04-30T20:35:59.551314Z level=debug logger=global msg="Printing at DEBUG level"
+ts=2020-04-30T20:35:59.551318Z level=info logger=scoped msg="Printing at INFO level"
+ts=2020-04-30T20:35:59.55132Z level=debug logger=scoped msg="Printing at DEBUG level"
+```
+
+By using `sigs.k8s.io/controller-runtime/pkg/log`, your logger is propagated through `controller-runtime`. Any logs produced by `controller-runtime` code will be through your logger, and therefore have the same formatting and destination.
+
+### Setting flags when running locally
+
+When running locally with `make run ENABLE_WEBHOOKS=false`, you can use the `ARGS` var to pass additional flags to your operator, including the zap flags. For example:
```

Comment:
I understand that we should suggest use the operator-sdk commands and not make one since shows that make cannot work in win SO. So, I think it would be `operator-sdk run --local --operator"ENABLE_WEBHOOKS=false"`

Reply:
We have been using `make` targets in the [quickstart](https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/kubebuilder/quickstart.md), hence using the same here.

Reply:
HI @joelanford, 

I understood that we should avoid use the make. Am I right? Should we no use the sdk commands here?